### PR TITLE
add object detection tutorial for Jetson

### PR DESCRIPTION
**Pull request by mseth10**
## Description ##
This PR adds an object detection tutorial using MXNet's v1.6 wheel with Jetson support.

### CODE REVIEW

**Comment by mseth10** [comment]
@samskalicky As per offline discussion, added 'GluonCV' in the title. Can you please review again?

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ctx = mx.gpu()
+
+# load model
+net = model_zoo.get_model('yolo3_darknet53_voc', pretrained=True, ctx=ctx)
+
+# load input image
+im_fname = utils.download('https://raw.githubusercontent.com/zhreshold/' +
+                          'mxnet-ssd/master/data/demo/dog.jpg',
+                          path='dog.jpg')
+x, img = data.transforms.presets.yolo.load_test(im_fname, short=512)
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:111]: GluonCV API

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+import mxnet as mx
+
+# set context
+ctx = mx.gpu()
+
+# load model
+net = model_zoo.get_model('yolo3_darknet53_voc', pretrained=True, ctx=ctx)
+
+# load input image
+im_fname = utils.download('https://raw.githubusercontent.com/zhreshold/' +
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:108]: GluonCV API

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+```python
+from gluoncv import model_zoo, data, utils
+from matplotlib import pyplot as plt
+import mxnet as mx
+
+# set context
+ctx = mx.gpu()
+
+# load model
+net = model_zoo.get_model('yolo3_darknet53_voc', pretrained=True, ctx=ctx)
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:105]: GluonCV API

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+im_fname = utils.download('https://raw.githubusercontent.com/zhreshold/' +
+                          'mxnet-ssd/master/data/demo/dog.jpg',
+                          path='dog.jpg')
+x, img = data.transforms.presets.yolo.load_test(im_fname, short=512)
+x = x.as_in_context(ctx)
+
+# call forward and show plot
+class_IDs, scores, bounding_boxs = net(x)
+ax = utils.viz.plot_bbox(img, bounding_boxs[0], scores[0],
+                         class_IDs[0], class_names=net.classes)
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:117]: GluonCV API

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+
+## Installing GluonCV and its dependencies
+
+We can install GluonCV on Jetson module using the following commands: 
+```bash
+sudo apt-get update
+sudo apt-get install -y python3-scipy python3-pil python3-matplotlib
+sudo apt autoremove -y
+sudo pip3 install gluoncv
+```
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:84]: This should go on the GluonCV installation page: https://gluon-cv.mxnet.io/install.html showing users how to install GluonCV on Jetson (ie. ARM)

File: docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+To complete this tutorial, you need:
+
+* A [Jetson module](https://developer.nvidia.com/embedded/develop/hardware) setup with [Jetpack 4.4](https://docs.nvidia.com/jetson/jetpack/release-notes/) installed using NVIDIA [SDK Manager](https://developer.nvidia.com/nvidia-sdk-manager)
+
+* Display (needed to view matplotlib plot) and keyboard setup to directly open shell on the module
+
+* [Swapfile](https://help.ubuntu.com/community/SwapFaq) installed, especially on Jetson Nano for additional memory (increase memory if the inference script terminates with a `Killed` message)
+
+## Installing MXNet v1.6 with Jetson support
```
Comment (samskalicky) [inline docs/python_docs/python/tutorials/deploy/inference/object_detection_jetson.md:48]: This whole section is duplicated from here: https://github.com/apache/incubator-mxnet/blob/master/docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md#installing-mxnet-v16-with-jetson-support We should just refer/link to it so we can keep the info in a single place. will be easier to keep updated and relevant

**Comment by mseth10** [comment]
Thanks for raising this concern @samskalicky . I agree with your suggestion to not have this tutorial as part of MXNet website, but to add it on Gluon-CV website.

@zhreshold do you think we can add this tutorial to gluon-cv website?

**Comment by mseth10** [comment]
I have created a PR to gluon-cv repo
https://github.com/dmlc/gluon-cv/pull/1346

**Comment by mseth10** [comment]
Thanks for agreeing to help with a preview for this website change @ys2843

**Comment by mseth10** [comment]
@samskalicky All above concerns are addressed. Can you please review again?

**Comment by ys2843** [comment]
> Thanks for agreeing to help with a preview for this website change @ys2843

Here is the link: http://ec2-34-219-134-42.us-west-2.compute.amazonaws.com/api/python/docs/tutorials/deploy/inference/index.html

**Comment by mseth10** [comment]
@mxnet-bot run ci [windows-cpu]

**Comment by mseth10** [comment]
@leezu can you please help merge this pr. thanks!

**Pull request merged**